wpeverest/user-registration
# UR-765 Feature - Trail Period for subscription.

### All Submissions:



### Changes proposed in this Pull Request:

- Introducing  `Trail Period for subscription` which allows user to register without payment until specified time after that recurring billing cycle will be started.

### Dependency PR -
User Registration Stripe - https://github.com/wpeverest/user-registration-stripe/pull/38

### How to test the changes in this Pull Request:

1. Create a form with subscription plan field and enable trail period.
2. Verify products are created in stripe dashboard or not.
3. Register user using subscription plan field.
4. Verify customer created and selected subscription plan assigned to customer or not and check billing cycle and invoices.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Trail Period for subscription.
